katsdpcontroller/product_controller.py

```diff
@@ -25,6 +27,7 @@
 
 BATCH_PRIORITY = 1        #: Scheduler priority for batch queues
 BATCH_RESOURCES_TIMEOUT = 7 * 86400   # A week
+CONSUL_URL = 'http://localhost:8500'
```

Comment:
That's something you made me do against my better judgement, since the whole point of consul is that it's always at a known endpoint to start service discovery. While it wasn't too painful to do for the master controller (which always runs in a fixed place) I really don't want to build a whole other service discovery mechanism for a product controller (when launched on some Mesos agent) to discover where to find its local consul agent.

Reply:
Indeed - I only noticed because you had pulled it out into a constant, which then seemed to cry out for a way to override it. Happy to leave as is...